src/shared-components/dropdown/mobileDropdown.js

```diff
@@ -13,15 +13,25 @@ const MobileDropdown = ({
       value={value || ''}
       onChange={onSelectChange}
     >
-      {options.map(option => (
-        <option
-          key={option.value}
-          value={option.value}
-          disabled={option.disabled}
-        >
-          {option.label}
-        </option>
-      ))}
+      {options.map(option => {
+        let disabledValue = option.disabled;
```

Comment:
Would this be clearer as `isDisabled` or `shouldDisable` (or `shouldBeDisabled`)? 

Also, should we change `option.disabled` to `!!option.disabled` to coerce undefineds to false?